src/main/java/org/openstreetmap/atlas/generator/creator/AtlasCreator.java

```diff
@@ -68,20 +78,27 @@ protected int onRun(final CommandMap command)
         final CountryBoundaryMap map = (CountryBoundaryMap) command.get(BOUNDARIES);
         final Shard tile = (Shard) command.get(TILE);
         final String pbfPath = (String) command.get(PBF_PATH);
-        final String shardingType = (String) command.get(SHARDING_TYPE);
-        final Sharding sharding = AtlasSharding.forString(shardingType, Maps.stringMap());
+        final String pbfScheme = (String) command.get(PBF_SCHEME);
+        final String pbfShardingName = (String) command.get(PBF_SHARDING);
+        final String shardingName = (String) command.get(SHARDING_TYPE);
+        final Sharding sharding = AtlasSharding.forString(shardingName, Maps.stringMap());
+        final Sharding pbfSharding = pbfShardingName != null
+                ? AtlasSharding.forString(shardingName, Maps.stringMap()) : sharding;
         final String countryName = (String) command.get(COUNTRY);
         final File output = (File) command.get(OUTPUT);
         PbfLoader.setAtlasSaveFolder(output);
-        final Atlas atlas = generateAtlas(map, tile, pbfPath, sharding, countryName);
-        atlas.save(output.child(countryName + "_" + tile.getName() + ".atlas"));
+        final Atlas atlas = generateAtlas(map, tile, pbfPath, pbfScheme, pbfSharding, sharding,
+                countryName);
+        atlas.save(output.child(countryName + CountryShard.COUNTRY_SHARD_SEPARATOR + tile.getName()
+                + FileSuffix.ATLAS));
         atlas.saveAsGeoJson(output.child(countryName + "_" + tile.getName() + ".geojson"));
```

Comment:
Nit: FileSuffix.Geo_JSON

Reply:
I can add this to another PR... Mind creating an issue for it?

Reply:
We can ignore this until later, this was already part of the existing implementation.